### DISCUSSION TITLE
Issue about Pytorch Geometric Temporal

### DISCUSSION THREAD

**Discussion by ajosemf** [pyg-team/pytorch_geometric#9249] [General]
Hey guys,

I registered an issue in the PyTorch Geometric Temporal repository and so far I have not had an answer. Could anyone here in this community give me an opinion on the issue?

Thanks in advance.

https://github.com/benedekrozemberczki/pytorch_geometric_temporal/issues/270

**Comment by rusty1s** [comment]
I am not that familiar with PyTorch Geometric Temporal :(

**Reply by ajosemf** [reply]
Hi @rusty1s ,

I'll wait, hoping someone can help me.

Anyway, thank you very much for the answer.
